backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+
+const users = {
+    "john_doe": { id: 1, username: "john_doe", email: "john@example.com" },
+    "jane_smith": { id: 2, username: "jane_smith", email: "jane@example.com" },
+    // Add more users as needed
+};
+
+router.get('/:username', (req, res) => {
+    const {username} = req.params;
+    const exists = username in users;
+    res.json({exists});
```

Comment:
i think we should wait for the database to be setup and the database user table written down to actually write this route to query for an existing user.